addons/mail/tests/common.py

```diff
@@ -120,3 +120,34 @@ def tearDownClass(cls):
         cls.env['ir.mail_server']._revert_method('build_email')
         cls.env['ir.mail_server']._revert_method('send_email')
         super(TestMail, cls).tearDownClass()
+
+    def gateway_reply_wrecord(self, template, record, extra_irt=True):
```

Comment:
Is it only used by tests or am I missing something?

Reply:
It is a test common file, stupid me.

Reply:
:D